utils/oscap.8

```diff
@@ -348,6 +348,12 @@ Takes component ref with given ID from checklists. This allows to select a parti
 .TP
 \fB\-\-benchmark-id ID\fR
 Selects a component ref from any datastream that references a component with XCCDF Benchmark such that its @id attribute matches given string exactly.
+.TP
+\fB\-\-tailoring-file TAILORING_FILE\fR
+Use given file for XCCDF tailoring. Needs to specify customized profile ID using --profile to take effect. If both --tailoring-file and --tailoring-id are specified, --tailoring-file takes priority.
```

Comment:
"Needs to specify Profile ID from tailoring file using --profile to apply."
"Use `--profile` to specify Profile ID from tailoring file to apply.

I think instructions should orient user to get Profile ID from Profiles in specified tailoring file, be it whatever it is, shadowed or not.

Reply:
"Select Profile from tailoring file to apply using --profile"

Reply:
+1